pmm-app/src/pmm-qan/panel/components/Overview/components/QanTable/Table.tsx

```diff
@@ -216,9 +221,16 @@ export const Table: FC<TableProps> = ({
           ) : null}
           {rows.length && !loading ? (
             <div {...getTableProps()} className="table">
-              <div {...getTableBodyProps()} className={cx('table-body', styles.tableBody(scroll.y))}>
+              <RSC
```

Comment:
If it's a component, then it should follow the naming convention, i.e. `ReactScrollbar`.